### ISSUE TITLE
Delete Projects and Companies

### ISSUE THREAD

**Issue by catherine-moser** [Leadera/ecoman_repo#257]
A consultant should also be able to delete HIS projects and companies. is this planned in the Admin-Pages?

**Comment by tcgumus** [comment]
Yes, also we need to talk about it together because it can cause some problems like companies can lost their cp and is data. So maybe we need to make it in admin pages or we should do it with an approval process.
